metricbeat/magefile.go

```diff
@@ -48,6 +47,8 @@ import (
 	"github.com/elastic/beats/v7/dev-tools/mage/target/unittest"
 	// mage:import
 	_ "github.com/elastic/beats/v7/dev-tools/mage/target/compose"
+	// mage:import
+	_ "github.com/elastic/beats/v7/metricbeat/scripts/mage/target/metricset"
```

Comment:
Should xpack/metricbeat/magefile.go get this too?

Reply:
Yes, I have added it there.